source/core/Console.cpp

```diff
@@ -602,7 +602,7 @@ namespace GTA
 	{
 		array<Script ^> ^scripts = ScriptDomain::CurrentDomain->RunningScripts;
 
-		if (scripts->Length == 0)
+		if (scripts->Length <= 1)
```

Comment:
I guess its probably too late to change now, but I dont think console should inherit the script class. It would also mean the console could be loaded into the default AppDomain before the ScriptDomain gets created and live on after it gets destroyed on insert key. Granted this would require a different way to call natives. most likely using natives.h file included in the SHV sdk. This implementation would also remove the nasty 2 class setup the console currently uses in favour of just having a single console class that handles everything(obviously the ConsoleArg/Command/DefaultCommand class would stay)